plugins/source/mixpanel/docs/tables/mixpanel_export_events.md

```diff
@@ -0,0 +1,16 @@
+# Table: mixpanel_export_events
+
+The composite primary key for this table is (**project_id**, **time**).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|project_id (PK)|Int|
+|time (PK)|Timestamp|
+|event|String|
```

Comment:
I don't know what `event` contains, but I think either that or one of the properties will need to be part of the PK as well: relying on `time` wouldn't be good enough to differentiate events, especially if it's only at secondly granularity, but even if it is at millisecond. Unless Mixpanel guarantees the timestamps to be unique?